src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.tsx

```diff
@@ -62,26 +66,38 @@ const PhoneAuthenticator: UISchemaElementComponent<{
     return `${code}${phone}`;
   };
 
+  const handleBlur = () => {
+    setIsTouched(true);
+    onValidateHandler(setFieldError, phone);
+  };
+
   useEffect(() => {
     onChangeHandler(formatPhone(phone, phoneCode, extension));
+    onValidateHandler(setFieldError, phone);
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [phoneCode, phone, extension, showExtension]);
 
+  // For server side errors, need to reset the touched value
+  useEffect(() => {
+    setIsTouched(false);
+  }, [error]);
+
   const renderExtension = () => (
     showExtension && (
-      <Box
-        width={0.25}
-      >
-        <TextInput
+      <Box width={0.25}>
+        <InputLabel htmlFor="phoneExtension">{getTranslation(translations, 'extension')}</InputLabel>
+        <OutlinedInput
+          value={extension}
           type="text"
-          data-se="extension"
           name="extension"
-          label={getTranslation(translations, 'extension')}
-          value={extension}
-          autocomplete="tel-extension"
-          onChange={(e: ChangeEvent<HTMLInputElement>) => {
+          id="phoneExtension"
+          onChange={(e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
             setExtension(e.currentTarget.value);
           }}
+          inputProps={{
+            'data-se': 'extension',
+            autocomplete: 'tel-extension',
```

Comment:
TIL how many `autocomplete` values there are that are allowed https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete